apps/src/sites/studio/pages/levels/_content.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+
+document.addEventListener(
+  'DOMContentLoaded',
```

Comment:
it seems like this might fire before this JS gets loaded, in which case your event handler would never fire, especially considering that DOMContentLoaded does not wait for script tags. I would suggest using `$(document).ready()` because it is guaranteed to run.